twoliter/embedded/rpm2kit

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+#
+# Create a kit from RPM package inputs.
+set -eu -o pipefail
+
+declare -a PACKAGES
+
+for opt in "$@"; do
+   optarg="$(expr "${opt}" : '[^=]*=\(.*\)')"
+   case "${opt}" in
+      --packages-dir=*) PACKAGES_DIR="${optarg}" ;;
+      --package=*) PACKAGES+=("${optarg}") ;;
+      --output-dir=*) OUTPUT_DIR="${optarg}" ;;
+   esac
+done
+
+# HACK: because we're not passing KIT from buildsys
+export KIT="${VARIANT}"
+
+# FIXME: should this be a versioned directory?
+# - needs to change whenever the kit changes?
+# - needs to be predictable for publishing to work?
+# - needs to avoid ambiguity if we have other local builds of the same kit?
```

Comment:
If we're emitting the right `cargo:rerun-if-changed` and `cargo:rerun-if-env-changed` directives in `buildsys` then we can drop this. Just need to make sure https://github.com/bottlerocket-os/twoliter/pull/198#discussion_r1594576970 is addressed.

Reply:
I think the answer is as simple as `rm -rf "${KIT_DIR}"` since, if this script is running, then we know we are re-creating the kit. That's what I've pushed.